src/views/viewer/ChannelView/ChannelView.style.tsx

```diff
@@ -4,12 +4,11 @@ import { fluidRange } from 'polished'
 import { ChannelLink } from '@/components/ChannelLink'
 import { TOP_NAVBAR_HEIGHT } from '@/components/TopbarBase'
 import { Button } from '@/shared/components/Button'
-import { IconButton } from '@/shared/components/IconButton'
+import { Search } from '@/shared/components/Search/Search'
```

Comment:
Should import from `@/shared/components/Search`. If `index.ts` doesn't exist there, let's add it